util.js

```diff
@@ -1,12 +1,29 @@
-const db = require("./db/connection.js")
+const db = require("./db/connection.js");
 
 exports.checkArticleExists = (article_id) => {
```

Comment:
If you look at these 2 functions they are pretty much doing the same thing. Have think about how you could refactor them into a single, slightly more dynamic, function to do the job of both and make your code a bit DRYer. To help make it easier you could send a more generic/less specific message - we don't have to be that kind to our client! (look in seminar slack channel there are some notes in there 👍 ).